pkg/commands/git.go

```diff
@@ -145,46 +195,11 @@ func (c *GitCommand) MergeStatusFiles(oldFiles, newFiles []File) []File {
 	return append(headResults, tailResults...)
 }
 
-func (c *GitCommand) verifyInGitRepo() {
-	if output, err := c.OSCommand.RunCommandWithOutput("git status"); err != nil {
-		fmt.Println(output)
-		os.Exit(1)
-	}
-}
-
 // GetBranchName branch name
 func (c *GitCommand) GetBranchName() (string, error) {
 	return c.OSCommand.RunCommandWithOutput("git symbolic-ref --short HEAD")
 }
 
-func (c *GitCommand) navigateToRepoRootDirectory() {
```

Comment:
I'm not sure why we're deleting this function. The purpose of this function is so that we if we start lazygit two directories deep into a repo, it will chdir back to the root so that we don't get any weird behaviour from running commands below the root level.

Reply:
It's a mistake, I forgot that git status works deep inside a repository, so I thought the first function that check if we are in a git repository was preventing the call of this function. I tested to see if it was the case yesterday and it wasn't but I probably mess up something by wanting to fit this wrong assumption. Sorry, I will fix it.